backend/alembic/versions/0034_virtual_collections_db_view.py

```diff
@@ -0,0 +1,306 @@
+                        r.path_cover_s as path_cover_s,
+                        r.path_cover_l as path_cover_l,
+                        jsonb_array_elements_text(igdb_metadata -> 'franchises') as collection_name,
+                        'franchise' as collection_type
+                    FROM roms r
+                    WHERE igdb_metadata->'franchises' IS NOT NULL
+                ),
+                collection_collection AS (
+                    SELECT
+                        r.id as rom_id,
+                        r.path_cover_s as path_cover_s,
+                        r.path_cover_l as path_cover_l,
+                        jsonb_array_elements_text(igdb_metadata -> 'collections') as collection_name,
+                        'collection' as collection_type
+                    FROM roms r
+                    WHERE igdb_metadata->'collections' IS NOT NULL
+                ),
+                modes_collection AS (
+                    SELECT
+                        r.id as rom_id,
+                        r.path_cover_s as path_cover_s,
+                        r.path_cover_l as path_cover_l,
+                        jsonb_array_elements_text(igdb_metadata -> 'game_modes') as collection_name,
+                        'mode' as collection_type
+                    FROM roms r
+                    WHERE igdb_metadata->'game_modes' IS NOT NULL
+                ),
+                companies_collection AS (
+                    SELECT
+                        r.id as rom_id,
+                        r.path_cover_s as path_cover_s,
+                        r.path_cover_l as path_cover_l,
+                        jsonb_array_elements_text(igdb_metadata -> 'companies') as collection_name,
+                        'company' as collection_type
+                    FROM roms r
+                    WHERE igdb_metadata->'companies' IS NOT NULL
+                )
+                SELECT
+                    collection_name as name,
+                    collection_type as type,
+                    'Autogenerated ' || collection_type || ' collection' AS description,
+                    NOW() AS created_at,
+                    NOW() AS updated_at,
+                    array_to_json(array_agg(rom_id)) as rom_ids,
+                    array_to_json(array_agg(path_cover_s)) as path_covers_s,
+                    array_to_json(array_agg(path_cover_l)) as path_covers_l
+                FROM (
+                    SELECT * FROM genres_collection
+                    UNION ALL
+                    SELECT * FROM franchises_collection
+                    UNION ALL
+                    SELECT * FROM collection_collection
+                    UNION ALL
+                    SELECT * FROM modes_collection
+                    UNION ALL
+                    SELECT * FROM companies_collection
+                ) combined
+                GROUP BY collection_type, collection_name
+                HAVING COUNT(DISTINCT rom_id) > 2
```

Comment:
Should this be `> 1` or `>= 2` instead?

Reply:
imo 2 games don't make a collection, so i set the minimum to 3 games